js/chatbot/shared/ChatBot.svelte

```diff
@@ -595,4 +596,18 @@
 			0 2px 2px rgba(0, 0, 0, 0.05);
 		transform: translateY(-2px);
 	}
+
+	.image-preview {
+		position: absolute;
+		z-index: 999;
+		left: 0;
+		top: 0;
+		width: 100%;
+		height: 100%;
+		overflow: auto;
+		background-color: var(--background-fill-secondary);
+		display: flex;
+		justify-content: center;
+		align-items: center;
+	}
```

Comment:
why is this css necessary? seemed to work fine before

Reply:
This is only needed for the full-screen image preview. Otherwise when you click on an image to pull up the full screen view it will be messed up

<img width="1232" alt="image" src="https://github.com/user-attachments/assets/3376024b-e83e-4c7f-b6ee-01dec65da57b">